app/src/main/java/com/hammer/talkbbokki/data/repository/UserInfoRepositoryImpl.kt

```diff
@@ -0,0 +1,35 @@
+package com.hammer.talkbbokki.data.repository
+
+import com.hammer.talkbbokki.data.local.DataStoreManager
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end